### ISSUE TITLE
No 'Access-Control-Allow-Origin' header is present

### ISSUE THREAD

**Issue by Gaurav0100** [pavelk2/social-feed#295]
Getting following error in fetching twitter feeds from yesterday. 

Failed to load https://api.jublo.net/codebird/oauth2/token: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'your website url' is therefore not allowed access. The response had HTTP status code 403.

I think this error is also coming on the demo page.

**Comment by rob-teamworks** [comment]
I'm getting the same issue on several client sites. Seems codebird is the problem?

**Comment by rob-teamworks** [comment]
Seems to be working now, maybe some service with Codebird was down and has just gone back up???

**Comment by Gaurav0100** [comment]
Still facing the same issue ...

**Comment by rob-teamworks** [comment]
Seems that it's an intermittent problem, it's now not working for me.

**Comment by thx1200** [comment]
Suddenly happening to me right now.  It seems codebird may be unreliable?  I see there are even issues from years ago exhibiting this issue intermittently.

**Comment by thx1200** [comment]
Oh well, this explains it.  I tried going to some of the codebird URLs just to see if I could get more details and it's throwing a CloudFlare generated "server is offline" message.  No wonder.

**Comment by CrySensai** [comment]
same here

Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin is therefore not allowed access. 

but with The response had HTTP status code **502**.

**Comment by thx1200** [comment]
Yes I'm having this problem again and it's been going on for I think for about a week now.  Before it would clear up pretty quickly, but now I'm wondering if this codebird api is just dead.

**Comment by Hypocrite** [comment]
Yeah, still dead.

**Comment by Gaurav0100** [comment]
@rob-teamworks again facing the same issue from last night.
